metal_header/sifive_gpio0.c++

```diff
@@ -197,6 +189,8 @@ void sifive_gpio0::define_inlines() {
         });
 
     count++;
+    if(count == num_gpios)
```

Comment:
Thanks for making the change. Can verify the case with two gpio nodes, each with one or more interrupts

Reply:
BTW, you Travis failed because of indent issue at line 193

Reply:
Thanks for verification. Format check passed, and indent issue resolved.